### ISSUE TITLE
'Skipping'element

### ISSUE THREAD

**Issue by Ar3s1110** [blacklanternsecurity/TREVORspray#20]
Hello,

First off I just wanted to say what a great tool and it does a whole lot more than just spraying.
My only question, and maybe I missed the flags. If I want to re use a password for spraying to see that it was successful in all green, how can I skip the "skipping" aspect of the spray. In other words, I dont want it to skip the password spray because it recognizes it was already used.

Thank you

**Comment by RapidWolf95** [comment]
Hey jrod1481!

There may be a better answer from someone more familiar with this, but I have found that there is a file (/home/[USERNAME]/.trevorspray/tried_logins.txt) containing a list of all the logins that have been attempted. I tested, and if I remove any relevant lines from there (or delete the whole file) a repeat attempt can be made.

For context, I am running on Kali and installed via pip per the install instructions in the readme. Your setup may be slightly different, but I would search for "tried_logins.txt" in the context of trevorspray.

Hopefully that helps answer your question.
